diagnostic.md

```diff
@@ -5,74 +5,83 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+This is a place to store the data and be able to support multiple users
+accessing the app.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
-```
+Controller```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+??
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
-```
+CREATE READ UPDATE DESTROY```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
-```
+GET
+POST
+PATCH
+PUT
+DELETE```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
-```
+API="${API_ORIGIN:-http://localhost:4741}"
+URL_PATH="/people/1"
+curl "${API}${URL_PATH}" \
+  --include \
+  --request GET \
+  --header "Authorization: Token token=$TOKEN"```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails new
 ```
 
 What is the command to start an instance of a rails server?
 
 ```bash
-// your response here
+bins/rails server
 ```
 
 What are the commands to drop, create, migrate and seed a database from the command
 line? (5 bullet points)
 
 ```bash
-// your response here
+db: drop
+db: create
+db: migrate
+db: seed
 ```
 
 What is the command to scaffold a pet with a name and age attributes (hint:
 Also think of the data types for each attribute)?
 
 ```bash
-// your response here
+rails generate scaffold Pet
```

Comment:
This is the right start, however you don't need to capitalize pet, also you forgot to include name:string and age:integer